packages/apps/tools/src/pages/transactions/transfer/index.tsx

```diff
@@ -81,6 +85,22 @@ const TransferPage = () => {
         paddingBlockEnd={'xxxl'}
         gap={'lg'}
       >
+        {!browserSupported ? (
```

Comment:
Should we also check whether the Ledger signing method is used to show this notification or not?